src/App.jsx

```diff
@@ -13,6 +14,7 @@ function App() {
         <Route path="/" element={<HomePage />} />
         <Route path="/signin" element={<SignInPage />} />
         <Route path="/signup" element={<SignUpPage />} />
+        <Route path="*" element={<NotFoundPage />} />
```

Comment:
'*' mean all, that mean wherever I go thro the website i'll get the NotFoundPage component.

Reply:
It will work when we set all the routes up, so he ddi that correctly.